lib/error_tracker/integrations/phoenix.ex

```diff
@@ -103,8 +103,8 @@ defmodule ErrorTracker.Integrations.Phoenix do
     })
   end
 
-  def handle_event([:phoenix, :live_view, :handle_event, :start], _, metadata, :no_config) do
-    ErrorTracker.set_context(%{
+  def handle_event([:phoenix, :live_view, :handle_event, :exception], _, metadata, :no_config) do
+    ErrorTracker.report({metadata.kind, metadata.reason}, metadata.stacktrace, %{
```

Comment:
I'd also remove the `[:phoenix, :live_view, :handle_event, :start]` from the list of events to subscribe to.

Reply:
Good catch! Should be fixed now.